backend/controller/observability/observability.go

```diff
@@ -0,0 +1,207 @@
+package observability
+
+import (
+	"context"
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/metric"
+	"time"
+
+	"github.com/alecthomas/types/optional"
+
+	"github.com/TBD54566975/ftl/backend/controller/dal"
+	ftlv1 "github.com/TBD54566975/ftl/backend/protos/xyz/block/ftl/v1"
+	"github.com/TBD54566975/ftl/backend/schema"
+	"github.com/TBD54566975/ftl/internal/log"
+	"github.com/TBD54566975/ftl/internal/model"
+)
+
+const name = "ftl.xyz/ftl/runner"
+
+type metricAttributeBuilders struct {
+	moduleName      func(name string) attribute.KeyValue
+	featureName     func(name string) attribute.KeyValue
+	destinationVerb func(name string) attribute.KeyValue
+}
+
+type callMetrics struct {
+	requests metric.Int64Counter
+	failures metric.Int64Counter
+	active   metric.Int64UpDownCounter
+	latency  metric.Int64Histogram
+}
+
+type fsmMetrics struct {
+	active      metric.Int64UpDownCounter
+	transitions metric.Int64Counter
+	failures    metric.Int64Counter
+}
+
+type observableMetrics struct {
+	meter      metric.Meter
+	attributes metricAttributeBuilders
+	calls      *callMetrics
+	fsm        *fsmMetrics
+}
+
+var (
+	metrics = observableMetrics{
+		meter: otel.Meter(name),
+		// TODO: move to a initialization method
```

Comment:
I might be missing something here, but based on how I read the spreadsheet for these, it seems like we would have a `meter` called something like `ftl.call` then that meter would contain counter/etc. like `requests` and `latency`. I'm still trying to make the names/columns in the spreadsheet to their counterparts here.

Reply:
change applied w/ the observability.go file decomposition into feature specific files.